loadgen/test_settings.h

```diff
@@ -153,6 +155,8 @@ struct TestSettings {
   uint64_t server_target_latency_ns = 100000000;
   /// \brief TODO: Implement this.
   bool server_coalesce_queries = false;
+  /// \brief The latency percentile for server mode. The default is for images models. It should be 97% for otherwise.
```

Comment:
suggested copyedit:

```
\brief The latency percentile for server mode. This value is combined with server_target_latency_ns to determine if a run is valid.
\details 99% is the default value, which is correct for image models. GNMT should be set to 0.97 (97%) in v0.5.
````

Reply:
Thanks for the copyedit. They will be fixed.